src/main/java/controller/RaceGame.java

```diff
@@ -0,0 +1,45 @@
+package controller;
+
+import objects.Car;
+import objects.Cars;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class RaceGame {
+    int gameCount;
+
+    private static Cars carRepository;
+
+    private static List<Car> winningCars = new ArrayList<>();
+
+    public Cars getCars(){
+        return carRepository;
+    }
+
+    public  RaceGame(String[] names , int gameCount){
```

Comment:
```suggestion
    public RaceGame(String[] names, int gameCount) {
```
사소하지만, 공백에 대한 컨벤션도 잘 지켜주세요!

Reply:
넵!! 수정했습니다